src/main/java/microsoft/exchange/webservices/data/core/EwsServiceMultiResponseXmlReader.java

```diff
@@ -98,7 +98,7 @@ private static XMLEventReader createXmlReader(InputStream stream)
    * @throws Exception on error
    */
   @Override
-  protected XMLEventReader initializeXmlReader(InputStream stream)
+  protected XMLEventReader initializeXmlReader(InputStream stream, Boolean forceXml11)
```

Comment:
maybe use `initializeXmlReader(..., boolean forceXml11)` instead of `initializeXmlReader(..., Boolean forceXml11)`